gads.py

```diff
@@ -17,41 +17,48 @@
 from flask import session
 from google.ads.googleads.client import GoogleAdsClient
 from fs_storage import get_value_session
+import logging
 
 
 def get_gads_client(mcc_id: str) -> GoogleAdsClient:
-  main_creds = json.loads(get_value_session(session['session_id'], 'credentials'))
-  creds = {
-      "developer_token": os.environ["GOOGLE_ADS_DEVELOPER_TOKEN"],
-      "refresh_token": main_creds["refresh_token"],
-      "client_id": main_creds["client_id"],
-      "client_secret": main_creds["client_secret"],
-      "use_proto_plus": True,
-  }
-
-  google_ads_client = GoogleAdsClient.load_from_dict(creds)
-  if mcc_id:
-    google_ads_client.login_customer_id = mcc_id
-
-  return google_ads_client
+  try:
```

Comment:
The try block should only be around the section of code that can fail instead of all the code. This will reduce the chance of catching another error and make debugging easier.

Reply:
Fixed